### PR TITLE
Remove blank line to get build to work

### PR DESCRIPTION
**Pull request by dbandrews**
Trying to get right new lines at end of `eazieda.py` for style check to pass in build.

### CODE REVIEW

**Review by arashshams — APPROVED** [review]

File: eazieda/eazieda.py [hunk 1]
```diff
@@ -119,16 +127,20 @@ def missing_imputed(data, impute=False, method_num="mean", method_non_num="most_
     impute : bool, default = False
         Whether to impute the missing values in place.
     method_num : str, default = "mean"
-        The method used for imputing numerical missing values. This is only applicable if impute=True.
+        The method used for imputing numerical missing values.
```
Comment (arashshams) [inline eazieda/eazieda.py:135]: Let's get rid of . at the end of docsrtings

File: eazieda/eazieda.py [hunk 2]
```diff
@@ -119,16 +127,20 @@ def missing_imputed(data, impute=False, method_num="mean", method_non_num="most_
...
         Whether to impute the missing values in place.
     method_num : str, default = "mean"
-        The method used for imputing numerical missing values. This is only applicable if impute=True.
+        The method used for imputing numerical missing values. 
+        This is only 
+        applicable if impute=True.
         One of 'drop', mean', 'median'
     method_non_num: str, default = "most_frequent"
-        The method used for imputing non-numerical missing values. This is only applicable if impute=True.
+        The method used for imputing non-numerical missing values.
```
Comment (arashshams) [inline eazieda/eazieda.py:140]: Let's get rid of . at the end of docsrtings

File: eazieda/eazieda.py [hunk 3]
```diff
@@ -119,16 +127,20 @@ def missing_imputed(data, impute=False, method_num="mean", method_non_num="most_
...
-        The method used for imputing non-numerical missing values. This is only applicable if impute=True.
+        The method used for imputing non-numerical missing values. 
+        This is only applicable if impute=True.
         One of 'drop', 'most_frequent'
 
     Returns
     -------
     pandas.core.frame.DataFrame
-        A dataframe contains two columns: the number of missing values and the percentage of missing values for each column
+        A dataframe contains two columns: the number of missing values and
```
Comment (arashshams) [inline eazieda/eazieda.py]: dataframe containing

Comment (dbandrews) [reply eazieda/eazieda.py]: Fixed.

File: eazieda/eazieda.py (hunk 2 above)
Comment (dbandrews) [reply eazieda/eazieda.py:140]: I think lets keep this in - it necessary information for now?

File: eazieda/eazieda.py (hunk 1 above)
Comment (dbandrews) [reply eazieda/eazieda.py:135]: I think this is useful information - let's keep in for this week?

**Pull request merged**